packages/xdl/src/detach/IosNSBundle.ts

```diff
@@ -36,7 +36,7 @@ const KERNEL_URL = 'https://expo.io/@exponent/home';
 
 function _configureInfoPlistForLocalDevelopment(config: any, exp: ExpoConfig): ExpoConfig {
   // add detached scheme
-  if (exp.isDetached && exp.detach.scheme) {
+  if (exp.isDetached && exp.detach && exp.detach.scheme) {
```

Comment:
we can use optional chaining now in this repo! so `exp.detach?.scheme` will be a better way to do this. same applies to some other things in this pr